### ISSUE TITLE
Pin upstream openshift-ansible

### ISSUE THREAD

**Issue by juddmaltin-dell** [redhat-openstack/openshift-on-openstack#234]
with the v0.8.0 release, the code is still recommending openshift-ansible/master.  Any thoughts on pinning it to a particular upstream branch?  branch enterprise-3.2?   tag v3.0.2-2 ?

**Comment by juddmaltin-dell** [comment]
Nevermind.  learned that openshift-ansible will be included in RPMs.  Need to ensure that OpenShift-on-OpenStack will use the RPMs.

**Comment by jprovaznik** [comment]
Yes, actually we want to pin to a stable tag of openshift-ansible for quite a long time now.  v3.0.2-2 sounds like a good candidate (supposing deployment works)

**Comment by juddmaltin-dell** [comment]
3.0.2-2 worked for me last week.  I haven't tried this week.  What would the process be of pinning this?  Making it the default in the openshift.yaml file?

**Comment by jprovaznik** [comment]
hm, so far I didn't have much luck with 3.0.2.-2, ansible failed quite quickly with:
fatal: [default-origin-master-0.example.com]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_name": "openshift_facts"}, "module_stderr": "OpenSSH_6.6.1, OpenSSL 1.0.1e-fips 11 Feb 2013\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 56: Applying options for *\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 30320\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\nTraceback (most recent call last):\n  File \"/tmp/ansible_E6Hgxc/ansible_module_openshift_facts.py\", line 992, in <module>\n    main()\n  File \"/tmp/ansible_E6Hgxc/ansible_module_openshift_facts.py\", line 973, in main\n    openshift_facts = OpenShiftFacts(role, fact_file, local_facts)\n  File \"/tmp/ansible_E6Hgxc/ansible_module_openshift_facts.py\", line 761, in **init**\n    self.system_facts = ansible_facts(module)\nTypeError: ansible_facts() takes exactly 2 arguments (1 given)\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 1\r\n", "module_stdout": "", "msg": "MODULE FAILURE", "parsed": false}

With master branch it didn't fail on this.
